### PR TITLE
Design doc for Orchestration

### PR DESCRIPTION
**Pull request by kumarashit**
Design doc for Orchestration

### CODE REVIEW

File: specs/capri/orchestration_automation_design.md [hunk 1]
```diff
@@ -0,0 +1,527 @@
...
+
+* **Connection Manager**: Connection Manager handles the workflow connector. It manages the connection between the OpenSDS Service Manager and the workflow manager. There can be different adapters which acts as workflow connector for OpenSDS workflow orchestration. Connector acts as an OpenSDS orchestration adapter for inbound and outbound integration with the Workflow Manager. In the current implementation, we use Stackstorm as the Workflow Manager and connector connects StackStorm to handle the interaction between service catalog manager and StackStorm.
+
+**StackStorm**: It’s an `IFTTT` framework which offers advanced automation, letting users create complex workflows to automate your world. Stackstorm breaks automation down into discrete actions and ties these actions together into workflows that can perform complex operations. StackStorm also provides a sophisticated event bus that can be used to implement event-based automation. Stackstorm enables OpenSDS Orchestration and Automation for the generic workflow actions based on OpenSDS projects such as hotpot, gelato and so forth.
+
+###Terminology
+
+ Service Catalog Manager
```
Comment (leonwanghui) [inline specs/capri/orchestration_automation_design.md]: Please add a `*` before each terminology.

File: specs/capri/orchestration_automation_design.md [hunk 2]
```diff
@@ -0,0 +1,527 @@
...
+                "input": {
+                  "cmd: "printf 'volume <% $.volue_id %> was attached to host successfully'"
+                }
+         }
+      }
+   }
+}
+```
+### Instance
+- Create service instance??will run immediately
```
Comment (leonwanghui) [inline specs/capri/orchestration_automation_design.md]: Why there are two `?` here?

File: specs/capri/orchestration_automation_design.md [hunk 3]
```diff
@@ -0,0 +1,527 @@
...
+      "host": {
+         "id": "string",
+         "ip": "10.0.0.2",
+         "host_name": "ubuntu001"
+      }
+   }
+ }
+```
+
+### Execution ?? move execution api to instance part
```
Comment (leonwanghui) [inline specs/capri/orchestration_automation_design.md]: Why there are two `?` here?

File: specs/capri/orchestration_automation_design.md [hunk 4]
```diff
@@ -0,0 +1,527 @@
...
+
+## Open discussion
+
+## References
+[1] Stackstorm
+https://docs.stackstorm.com/
+
+## Addendum
+Sample provisioning yaml file:
+---
```
Comment (leonwanghui) [inline specs/capri/orchestration_automation_design.md]: Please use ```yaml to format the yaml file.

File: specs/capri/orchestration_automation_design.md (hunk 1 above)
Comment (kumarashit) [reply specs/capri/orchestration_automation_design.md]: Done

File: specs/capri/orchestration_automation_design.md (hunk 2 above)
Comment (kumarashit) [reply specs/capri/orchestration_automation_design.md]: Corrected

File: specs/capri/orchestration_automation_design.md (hunk 3 above)
Comment (kumarashit) [reply specs/capri/orchestration_automation_design.md]: Corrected

File: specs/capri/orchestration_automation_design.md (hunk 4 above)
Comment (kumarashit) [reply specs/capri/orchestration_automation_design.md]: Done
Thanks

**Comment by xing-yang** [comment]
@kumarashit Do you still need https://github.com/opensds/design-specs/pull/30?  Please close the other one since you've opened another PR here.  Thanks.

**Pull request merged**